src/lib/git.js

```diff
@@ -174,6 +174,16 @@ async function sha(dir, branch) {
   return await git(dir, "show-ref", "-s", `refs/remotes/origin/${branch}`);
 }
 
+/**
+ * It retrieves the hash from a remote repository and branch
+ *
+ * @param {string} repositoryUrl the repository URL
+ * @param {string} branch the branch to get the hash from
+ */
+async function remoteSha(repositoryUrl, branch) {
+  return await git(".", "ls-remote", repositoryUrl, branch);
```

Comment:
@almope do you think there is a better way to get hash from a remote repository? :thinking: thanks!